content/en/docs/languages/js/propagation.md

```diff
@@ -7,7 +7,7 @@ cSpell:ignore: rolldice
 
 {{% docs/languages/propagation js %}}
 
-## Automatic context propagations
+## Automatic context propagation
```

Comment:
I assume js-approvers were tagged for this change. This looks fine. :)

Reply:
Haha, yeah! Theirs is already approved / reviewed :)